### ISSUE TITLE
Add support for "Database maintenance"

### ISSUE THREAD

**Issue by XTChrys** [keepassxreboot/keepassxc#7550]
Hi developers! As the title suggests, is it possible to add support for database maintenance like Keepass? Thank you in advance! :)

Example:
![dbm](https://user-images.githubusercontent.com/61419802/159114994-261d26b2-8aba-4cc5-bb16-72872e15b779.jpg)

**Comment by droidmonkey** [comment]
Yes in fact we added a database maintenance section to the database settings in 2.7.0. It only purges unused icons right now.

**Comment by XTChrys** [comment]
Ok! Thanks for the reply! :)

**Comment by Kache** [comment]
I'm a little confused, https://keepassxc.org/docs/KeePassXC_UserGuide#_database_maintenance
says there should be a feature to purge unused icons, but I don't seem to have this feature

Is it not available on the mac build or something?

**Comment by droidmonkey** [comment]
Are you running version 2.7.6? Are you looking in the database settings?

**Comment by Kache** [comment]
version 2.7.6
database settings: with the database open, I click on the gear in the icon toolbar in between the dice (pw generator) and search box.

I'd take a screenshot, but I think keepassxc doesn't let me do it. In screenshots it's as if it were 100% transparent.

My settings UI also differs from the [screenshot](https://keepassxc.org/docs/KeePassXC_UserGuide#_database_maintenance) in a few ways:
 * an "Application Settings" title below the icon toolbar and left settings icons
 * my left setting icons are as follows: general, security, browser integration, ssh agent, keeshare, whereas the screenshot shows general, security, keeshare, browser integration, maintenance
 * I don't have an "Advanced Settings" checkbox at the bottom left

**Comment by droidmonkey** [comment]
That is the application settings. Go to Database menu on the top toolbar of macos and choose Database settings. Or use the keyboard shortcut `Cmd+Shift+,`

**Comment by ygguser** [comment]
There are other useful items in KeePass.:

- Delete Dublicate Entries
- Delete Empty Ggoups
- Delete Unused Custom Icons

<img width="738" height="328" alt="Image" src="https://github.com/user-attachments/assets/41f646a5-fc9a-4650-bac8-91741906d807" />

It would be good if you could implement this as well.
